open-company/open-company-web
# add image as existing url

[trello](https://trello.com/c/TTRanxyC/414-add-image-as-existing-url)

To test:

- edit any post
- move cursor to an empty line
- press (+)
- choose "provide URL"
- paste a URL to any image you like
- hit "add

The image you provided should be uploaded with progress reporting and ultimately be added to the post.

In the background this adds a marker element to the post so that we can replace that marker with the uploaded image later.
